datalad_service/common/s3.py

```diff
@@ -44,27 +44,27 @@ def s3_url(self):
 def setup_s3_sibling(dataset, realm):
     """Add a sibling for an S3 bucket publish."""
     dataset_id = os.path.basename(dataset.path)
-
-    if (realm == DatasetRealm.PUBLIC):
+    annex_options = [
+        'type=S3',
+        'bucket={}'.format(realm.s3_bucket),
+        'exporttree=yes',
+        'partsize=1GiB',
+        'encryption=none',
+        'fileprefix={}/'.format(dataset_id),
+    ]
+    if realm == DatasetRealm.PUBLIC:
         public = getattr(datalad_service.config, 'DATALAD_S3_PUBLIC_ON_EXPORT')
```

Comment:
- I was not sure what is this additional configuration in addition to the 'realm' above about. Pardon me if I misunderstood it as just as an ability to override
- note that this condition block is not covered by tests, that is why coverage decreased while I moved setting of public remote options within this block

Reply:
There is one deployment of OpenNeuro which does not want the public bucket objects to be available outside of their organization, the DATALAD_S3_PUBLIC_ON_EXPORT is for that. On OpenNeuro.org it is always true.

Reply:
so it is for the override.  I would have though did that check of config outside of this function call, or made `realm=None` to be the default and then consult the config if it is None, defaulting to False.  I.e. by default the export would not be public, if explicitly specified - then you get it public.